src/components/Form.jsx

```diff
@@ -35,7 +39,12 @@ function Form() {
         </label>
       </div>
       <div className="btnContainer">
-        <button className="btnBottle" type="button">Ajouter une autre bouteille</button>
+        <button
+          className="btnBottle"
+          type="button"
+        >
+          { link('/login', 'Se connecter') }
+        </button>
```

Comment:
no button in a link: put a button or a link, then apply the style you want with css ;)